configure.sh

```diff
@@ -212,7 +123,7 @@ build:nohdfs --define=no_hdfs_support=true
 build:nonccl --define=no_nccl_support=true
 
 build --config=v2 --config=noaws --config=nogcp --config=nohdfs --config=nonccl
-test --config=v2 --config=noaws --config=nogcp --config=nohdfs --config=nonccl
+test --config=v2
```

Comment:
What does this do?

Reply:
This was added by mistake and will throw warnings since it is already configured on the build. This makes sure we don't build TensorFlow with those features since we don't need them.